src/libecalc/presentation/simple_result/simple.py

```diff
@@ -133,6 +134,10 @@ class SimpleComponentResult(SimpleBase):
     energy_usage_unit: Unit
     power: Optional[List[opt_float]]
 
+    @classmethod
+    def from_dto(cls, component_result) -> "SimpleComponentResult":
```

Comment:
typehint

Reply:
So a protocol then? BaseModel from pydantic? Import type from dto.results?

Reply:
Nevermind, I was thinking simple shouldn't import from dto, but that is what I'm fixing here..